lib/appsignal/check_in.rb

```diff
@@ -40,6 +48,35 @@ def cron(identifier)
         output
       end
 
+      # Track heartbeat check-ins.
+      #
+      # Track the execution of long-lived processes by sending a heartbeat
+      # check-in.
+      #
+      # @example Send a heartbeat check-in
+      #   Appsignal::CheckIn.heartbeat("main_loop")
+      #
+      # @param identifier [String] identifier of the heartbeat check-in to report.
+      # @yield the block to monitor.
+      # @return [void]
+      # @since 4.1.0
+      # @see https://docs.appsignal.com/check-ins/heartbeat
+      def heartbeat(identifier, forever: false)
```

Comment:
Was the name forever decided upon yet? We can also name it continuous heartbeats or something. Forever seems so definitive.

Reply:
> It's not at all set in stone. Something like `continously: true` sounds good to me as well.

My vote is for something other than forever, like `repeat: true` or `continuous: true`.

Reply:
Went with `continuous: true`.